apps/cli/src/domain/node.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
+
+import { Dependencies } from "./dependencies.js";
+import { unwrapOrLogError } from "./index.js";
+
+const ScriptName = z.string().brand<"ScriptName">();
+
+export const ScriptSchema = z.object({
+  name: ScriptName,
+  script: z.string(),
+});
+
+export type Script = z.infer<typeof ScriptSchema>;
+
+export interface NodeReader {
+  getScripts(cwd: string): Promise<Script[]>;
+}
+
+const requiredRootScripts = ["code-review"];
+
+export const checkMonorepoScripts =
+  (monorepoDir: string) =>
+  async (dependencies: Pick<Dependencies, "logger" | "nodeReader">) => {
+    const { logger, nodeReader } = dependencies;
+
+    const scripts = await unwrapOrLogError(dependencies)(() =>
+      nodeReader.getScripts(monorepoDir),
+    );
+    requiredRootScripts.map((script) => {
+      const exists = scripts.some(({ name }) => name === script);
+      if (exists) {
+        logger.success(`Script "${script}" is present in the monorepo root`);
+      } else {
+        logger.error(`Script "${script}" is missing in the monorepo root`);
+      }
+    });
```

Comment:
I am not sure, but maybe this check can be done with `zod` by parsing the `scripts` by the `requiredScript` schema (like `({ "code-review": z.string().nonempty()`)